### DISCUSSION TITLE
Achievement bug

### DISCUSSION THREAD

**Discussion by Turtlebole** [community/community#31223] [Other Feature Feedback, Questions, & Ideas] [answered]
I noticed that I unlocked 3 of the beta achievements, but only 2 of them are displaying on my profile, while when the Achievement field is clicked it shows that I have 3 of them. I am unsure if this is a bug, did anyone else experience similar things?

![gitbug](https://user-images.githubusercontent.com/96013970/187192503-5d8a1803-c470-4e1d-bba7-f17f7925c8f4.png)

**Comment by melvincwng** [comment, marked as answer]
Hi @Turtlebole, based on what I've seen from your profile, I have two suggestions - please feel free to try out either one:

1. I noticed the 'QuickDraw' achievement is missing. This could be because in your PR over [here](https://github.com/Turtlebole/Turtlebole/pull/1), it could be because the PR was closed and not merged. Try merging another PR within 5 mins instead of closing it, and see if the achievement reappears.

2. If that doesn't work, it could be a bug. In that case, you would have to raise a ticket to [GitHub Bug Report](https://support.github.com/contact/bug-report). Explain to them the situation, and provide them at least a few working days to investigate and resolve the issue.

Hope this answers your question, take care 😄

**Reply by Turtlebole** [reply]
Thank you for the options, tried merging another PR within 5 minutes but it didn't work, maybe I did something wrong, I am actually unsure 😅, it could be a visual bug.

**Reply by melvincwng** [reply]
@Turtlebole, in that case, I would highly suggest you submit a ticket to GitHub Bug Report - https://support.github.com/contact/bug-report

They usually respond pretty quickly, and if it's really an issue on their end, I'm sure you'll get your badge very soon :)

Take care, and good luck!
